lib/commandCenter/commandCenter.js

```diff
@@ -670,6 +670,7 @@ CommandCenter.prototype.executeBatch = function (acceptedEncodings, batch, query
 	var state = new State();
 
 	state.appName = this.app.name;
+	state.data = metaData;
```

Comment:
If no meta data is provided, will `metaData` still be an empty object? `state.data` defaults to an empty object, and I'm guessing we wouldn't want to overwrite that with `null` or `undefined`. At the user command level, there's an if-statement around the assignment:

https://github.com/stelcheck/mage/blob/a8a0faa1d74e3596ccf2316a195e781ec866f16c/lib/commandCenter/commandCenter.js#L495-L497